app/components/public/sponsor-list.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+import { orderBy, groupBy } from 'lodash';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  init() {
+    this._super(...arguments);
+    this.set('sponsors', groupBy(orderBy(this.get('sponsors'), 'level'), 'type'));
```

Comment:
Please add this as a computed property. Otherwise, this will not change incase sponsors array changes.